### PR TITLE
SAK-31636 Rename existing 'Home' tools

### PR DESCRIPTION
**Pull request by marktriggs**

### CODE REVIEW

File: reference/docs/conversion/sakai_12_mysql_conversion.sql [hunk 1]
```diff
@@ -126,3 +126,9 @@ create unique index dash_lock_ts_idx on dash_task_lock (task, server_id);
 INSERT INTO SAKAI_SITE_PAGE VALUES('!user-99', '!user', 'Dashboard', '0', 0, '0' );
 INSERT INTO SAKAI_SITE_TOOL VALUES('!user-999', '!user-99', '!user', 'sakai.dashboard', 1, 'Dashboard', NULL );
 
+
+--
+-- SAK-31636 Rename existing 'Home' tools
+--
+
+update sakai_site_page set title = 'Overview' where title = 'Home';
```
Comment (buckett) [inline reference/docs/conversion/sakai_12_mysql_conversion.sql]: Should be `update SAKAI_SITE_PAGE set title = 'Overview' where title = 'Home';` as us silly MySQL people have case sensitive databases.

**Comment by buckett** [comment]
Is it worth putting this in the 11.1 upgrade?

**Comment by marktriggs** [comment]
Ah, thanks.  I've adjusted the case for MySQL now, and added new conversion files for 11.1.

**Comment by steveswinsburg** [comment]
Conflicts 😭  - sorry I think that is because I just merged another PR into the conversion scripts 🐴

**Comment by marktriggs** [comment]
Heheh no problem.  As per Standard Australian Protocol I slept a random amount of time and retried, so hopefully we don't collide again :)

**Comment by steveswinsburg** [comment]
Struth, she'll be right, flaming galahs at Github! 🐨

**Pull request merged**